src/porepy/models/solution_strategy.py

```diff
@@ -173,6 +211,31 @@ def initial_condition(self) -> None:
         vals = np.zeros(self.equation_system.num_dofs())
         self.equation_system.set_variable_values(vals, to_iterate=True, to_state=True)
 
+        # Overwrite states from file if restart is enabled.
+        if self.restart_options.get("restart", False):
+            if "pvd_file" in self.restart_options:
```

Comment:
I think the default `restart_option` will have `"pvd_file"` as a key, but with a `None` value.

Reply:
You are right. I changed to a slightly different conditional, testing for both its presence (if the restart options are defined by the user wanting to restart from vtu files, it may happen that the keyword pvd_file is not provided, or set to None, which are both OK here.)